BlackNectar/BlackNectar/AppDelegate.swift

```diff
@@ -49,6 +54,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
         
     }
+    
+    func logUser() {
+        
+        Crashlytics.sharedInstance().setUserIdentifier(UIDevice.current.name)
```

Comment:
I don't know if this is a really useful identifier, but better than nothing I guess.